sdk/src/system_instruction.rs

```diff
@@ -38,6 +45,20 @@ pub enum SystemInstruction {
         space: u64,
         program_id: Pubkey,
     },
+    /// Create a new account at an address derived from
+    ///    the from account and a seed
+    /// * Transaction::keys[0] - source
+    /// * Transaction::keys[1] - new account key
+    /// * seed - string of ascii chars, no longer than MAX_ADDRESS_SEED_LEN
+    /// * lamports - number of lamports to transfer to the new account
+    /// * space - memory to allocate if greater then zero
+    /// * program_id - the program id of the new account
+    CreateAccountWithSeed {
```

Comment:
We'll need to adjust solana-web3.js once this lands, do you want a hand with that @rob-solana?

Reply:
lemme take a crack first

Reply:
ok, I failed: https://github.com/solana-labs/solana-web3.js/pull/571